### DISCUSSION TITLE
Remove the back side of many images

### DISCUSSION THREAD

**Discussion by miaacostach** [developersharif/bgremover-app#4] [General]
I need to move the background of many images, this tool works very well with a single image. I don't know if it can be done in bulk.

or some guide to do it, because I have read about convolutional network in py but that is still very complicated for me.

all the images are like this, but with different seeds colors. The background is the same.

![G  275 seed](https://user-images.githubusercontent.com/90373455/226721455-7e0587ac-b3f9-4a33-9492-89e3376d4d4e.jpg)
![bgremover_tmpu9bz81wc](https://user-images.githubusercontent.com/90373455/226721865-c5c520b8-3975-4434-80f1-e1b69f9211c0.png)

**Comment by developersharif** [comment]
Thank you for using our background remover tool! Currently, our tool only supports removing the background of a single image at a time. However, we are working on adding the feature to remove the background of multiple images in bulk in the next version.

**Reply by miaacostach** [reply]
Thank you for fast response. This is an excellent tool for the users that we are not basic programmers but we like this topic very much. I have 1500 images to remove the background and so with R libraries extract the values of each pixel for other analysis.
